src/pages/docs/worker-bridge.mdx

```diff
@@ -0,0 +1,452 @@
+These definitions allow you to expose endpoints to users and  specify how incoming requests should be managed by your `Golem` worker workflow/application.
+
+Let's dig a bit deeper using an example.
+
+Say you already have a worker running in your `Golem OSS` (docker environment) or `Golem Cloud`.  Assuming you have already got the worker running in `Golem` with shopping-cart example,
+let's say we have to expose an endpoint to get the cart contents for a user.  In order to do this, you would typically write an API definition as given below and simply upload to `Golem`.
+
+
+At this point, worker-bridge is acting as an API-Gateway, but the major difference here is, it is geared
+to work seamlessly with `Golem`. In other words, we don't need to specifically write a custom backend service as worker-bridge gives it for free.
+
+Here is a simple API definition. You have to replace with the correct template-id and function name once you update your shopping-cart template to golem. Consider the above
+as a simple example.
+
+```json
+
+{
+  "id": "shopping-cart-v1",
+  "version": "0.0.3",
+  "routes": [
+    {
+      "method": "Get",
+      "path": "/{user-id}/get-cart-contents",
+      "binding": {
+        "type": "wit-worker",
+        "template": "2696abdc-df3a-4771-8215-d6af7aa4c408",
+        "workerId": "worker-${request.path.user-id}",
+        "functionName": "golem:it/api/get-cart-contents",
+        "functionParams": [],
+        "response" : "${worker.response}"
+      }
+    }
+  ]
+}
+
+```
+
+
+Let's break this down.
+
+**id**: This field represents the unique identifier for the API definition. In this case, it is set to "shopping-cart-v1".
+
+**version**: This field indicates the version of the API definition. Here, it is set to "0.0.3".
+
+**routes**: This field contains an array of route objects, each representing a specific endpoint definition.
+
+**method**: Indicates the HTTP method associated with the route. In this example, it is set to "Get", indicating that this route handles GET requests.
+
+**path**: Specifies the URL path pattern for the route. It may include path parameters enclosed in curly braces. Here, the path is "/{user-id}/get-cart-contents", indicating that this route handles requests to retrieve the contents of a shopping cart for a specific user.
+
+**binding**: This object contains information about how the request should be handled by the Golem worker.
+
+### Golem Worker Binding
+
+Let's break down the binding object.
+
+**type**: Specifies the type of binding. Here, it is set to "wit-worker", indicating that the binding involves a Golem worker. Currently, the only supported type is "wit-worker".
+
+**template**: Provides the template ID associated with the worker binding.
+
+**workerId**: Specifies the ID of the Golem worker responsible for handling the request. The value is wrapped in code block starting with `${` and ending with `}`. Anything wrapped in this block is an `expression`.
```

Comment:
A common pattern would be to want a new (random) id for each request.

This should be documented. I think workerId should be optional, and if missing, we generate a UUID for the worker id.

Reply:
You might also suggest they could use correlation id, trace id, or something similar to tag each new request with a new worker.